node/network/availability-recovery/src/tests.rs

```diff
@@ -41,13 +41,13 @@ struct TestHarness {
 fn test_harness<T: Future<Output = ()>>(
 	test: impl FnOnce(TestHarness) -> T,
 ) {
-	let _ = env_logger::builder()
-		.is_test(true)
-		.filter(
-			Some("polkadot_availability_recovery"),
-			log::LevelFilter::Trace,
-		)
-		.try_init();
+	// let _ = env_logger::builder()
+	// 	.is_test(true)
+	// 	.filter(
+	// 		Some("polkadot_availability_recovery"),
+	// 		log::LevelFilter::Trace,
+	// 	)
+	// 	.try_init();
```

Comment:
some uncommented code (I personally find env_logger useful, as it only shows logs for failed tests)